tests/functional/context_methods/test_builtin_functions.py

```diff
@@ -105,26 +109,18 @@ def test_builtin_invocation_args_dict_function(self, project):
 
         parsed_logs = parse_json_logs(log_output)
         result = find_result_in_parsed_logs(parsed_logs, "invocation_result")
-
         assert result
-
-        # Result is checked in two parts because profiles_dir is unique each test run
-        expected = "invocation_result: {'debug': True, 'log_format': 'json', 'write_json': True, 'use_colors': True, 'printer_width': 80, 'version_check': True, 'partial_parse': True, 'static_parser': True, 'profiles_dir': "
-        assert expected in str(result)
-
+        # The result should include a dictionary of all flags with default values that aren't None
         expected = (
-            "'send_anonymous_usage_stats': False",
```

Comment:
`send_anonymous_usage_stats` is not working as expected, see comments in https://github.com/dbt-labs/dbt-core/commit/da9ae7f4bb76c7c1674ed0703d98c448c1ca3abb

Reply:
should we add this back in, and expect to see the test pass once the issues with `send_anonymous_usage_stats` are resolved?

Reply:
sure! I'll add it back in, and we can expect that the fix to `send_anonymous_usage_stats` will resolve this test as well